tool/teleport/common/teleport.go

```diff
@@ -299,6 +299,7 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 	dump.Flag("auth-server", "Address of the auth server.").StringVar(&dumpFlags.AuthServer)
 	dump.Flag("app-name", "Name of the application to start when using app role.").StringVar(&dumpFlags.AppName)
 	dump.Flag("app-uri", "Internal address of the application to proxy.").StringVar(&dumpFlags.AppURI)
+	dump.Flag("node-labels", "Labels to add to newly created nodes").StringVar(&dumpFlags.NodeLabels)
```

Comment:
I would definitely mention comma-separated and add an example to the help message here.